test/iron-image.html

```diff
@@ -26,6 +46,19 @@
         <iron-image></iron-image>
       </template>
     </test-fixture>
+
+    <test-fixture id="FixedWidthContainer">
+      <template>
+        <div class="fixed-width-container"></div>
```

Comment:
For these fixtures, you should go ahead and include the `iron-image` in the DOM. This way you only need to stamp one fixture, and you don't have to do any manual reconstruction to get the DOM into the shape you want it to be in for the test.